db/migrate/20150618183029_add_extensions_to_versions.rb

```diff
@@ -0,0 +1,5 @@
+class AddExtensionsToVersions < ActiveRecord::Migration
+  def change
+    add_column :versions, :extensions, :boolean
```

Comment:
We can definitely go through all of the gemspecs. We have before and it's not a huge deal...just takes a while. We don't have millions of gems, either:

```
irb(main):001:0> Version.indexed.count
=> 597787
```

Basically we shouldn't be afraid of tackling problems that force us to download a few hundred thousand files. In the grand scheme of things we're going to make things better for everyone in the Ruby community when we take steps like that.